core/tests/protractor_utils/ExplorationEditorSettingsTab.js

```diff
@@ -109,24 +113,28 @@ var ExplorationEditorSettingsTab = function() {
       closePreviewSummaryButton);
     await waitFor.invisibilityOf(
       explorationSummaryTile, 'Summary Tile takes too long to disappear');
-    await action.click('Neutral element', neutralElement);
+    await action.click('Neutral element', neutralElement, true);
   };
 
   this.setCategory = async function(category) {
+    await action.click('Neutral element', neutralElement);
```

Comment:
Why is this needed now when it wasn't necessary before?

Reply:
removed